Sources/AWSLambdaRuntimeCore/ControlPlaneRequest.swift

```diff
@@ -51,14 +51,10 @@ struct Invocation: Hashable {
             throw Lambda.RuntimeError.invocationMissingHeader(AmazonHeaders.invokedFunctionARN)
         }
 
-        guard let traceID = headers.first(name: AmazonHeaders.traceID) else {
-            throw Lambda.RuntimeError.invocationMissingHeader(AmazonHeaders.traceID)
-        }
-
         self.requestID = requestID
         self.deadlineInMillisSinceEpoch = unixTimeInMilliseconds
         self.invokedFunctionARN = invokedFunctionARN
-        self.traceID = traceID
+        self.traceID = headers.first(name: AmazonHeaders.traceID) ?? ""
```

Comment:
should this emit a warning in the case its missing? wdyt @fabianfett ?

Reply:
I think, we should just add a self created traceID. We already have a function for this:
```suggestion
        self.traceID = headers.first(name: AmazonHeaders.traceID) ?? 
            "Root=\(AmazonHeaders.generateXRayTraceID());Sampled=0"
```